### DISCUSSION TITLE
Webhooks plugin

### DISCUSSION THREAD

**Discussion by NevRA** [saleor/saleor#10015] [Q&A] [answered]
Hey!
Could you please explain the purpose of the Webhooks Plugin, do I need it on the production?
For example, if I have Stripe integration via the Stripe plugin, and configured Stripe webhook there, why do I need another plugin?

From what I see in the implementation Webhooks Plugins sends all registered requests like 'order_created' back to the app but I can't find it in the documentation.

Thanks!

**Comment by timuric** [comment, marked as answer]
Hi @NevRA you don't need this plugin, Saleor supports many webhooks natively, simply use the Apps section in the dashboard to configure webhooks.
